hadoop-ozone/dist/src/main/dockerbin/entrypoint.sh

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env bash
+##
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+##
+set -e
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+if [ -n "$SLEEP_SECONDS" ]; then
+   echo "Sleeping for $SLEEP_SECONDS seconds"
+   sleep $SLEEP_SECONDS
```

Comment:
shellcheck:10: note: Double quote to prevent globbing and word splitting. [SC2086]

Reply:
You might need pwd -P to resolve symlinks.  I don't recommend to use script location to make decision of where binaries are supposed to be because someone else can make newbie mistake and refactor your script to invalid the original coding intend.  See this blog to explain [the right way to get the directory of a bash script|https://www.ostricher.com/2014/10/the-right-way-to-get-the-directory-of-a-bash-script/].  This is the reason that I used OZONE_HOME as base reference frequently.

Reply:
Thanks the comment @eyanghwx

Usually I prefer to avoid significant changes during a move because it won't be separated from the copy/move:
 * It can be harder to follow what has been changed
 * And it can be harder to revert in case of any problem

I created an issue to track this problem:

https://issues.apache.org/jira/browse/HDDS-1642